governance.md

```diff
@@ -0,0 +1,164 @@
+In practice, it is different from a majority-vote-wins approach.
+In the Op1st project a discussion toward a decision follows this process:
+
+1. A proposal is put forth and a check for consensus is made.
+   1. Consensus is signified through a +1 vote.
+1. A check is made for any dissent on the proposal.
+   1. Reservations? State reservation, sometimes with a ‘-1’ signifier
+      1. Reservations about the proposal are worked through, seeking consensus to resolve the reservations.
+      2. A reservation is not a vote against the proposal, but may turn into a vote against if unresolved.
+      It is often expressed with an initial -1 vote to indicate reservations and concerns.
+      This indicates there is still discussion to be had.
+   1. Stand aside?
+      No comment, or state concerns without a -1 reservation; sometimes the ‘-0’ signifier is used.
+      1. This option allows a member to have issues with the proposal without choosing to block the proposal, by instead standing aside with a +/-0 vote.
+      1. The stated concerns may influence other people to have or release reservations.
+   1. Block?
+   Vote ‘-1’ with reasons for the block.
+     1. This is a complete block on a proposal, refusing to let it pass.
+     A block is a -1 vote and must be accompanied with substantive arguments that are rooted in the merit criteria of the Project – protecting the community, the upstream, technical reasons, and so forth.
+
+
+### Use of Architectural Decision Records (ADR)
+Architectural decisions are design decisions with some level of architectural significance.
+At Op1st, we track architectural decisions using lightweight architectural decision records.
+More information on the format we follow is available at https://adr.github.io/madr/.
+
+ADR page: https://www.operate-first.cloud/blueprints/blueprint/
+
+ADR repository: https://github.com/operate-first/blueprint
+
+### Use of the Community Handbook
+The Community Handbook is intended to serve as a resource for all Operate First community contributors.
+It contains documentation on how to perform contributor-related tasks and encourages input and contributions from the community.
+The covenant of the Community Handbook is that if contributors must figure out a task for themselves, they should submit that solution to the handbook for the benefit of future contributors.
+
+Content for the Community Handbook is drawn from all across the project, allowing documentation to live close to the place it's used.
+There is a core repository for structure, guidelines, and all content that does not fit or belong in another repository.
+
+Community Handbook page: https://www.operate-first.cloud/community-handbook/
+
+Community Handbook repository: https://github.com/operate-first/community-handbook
+
+## Governance roadmap: 1.1 and beyond
+
+This section contains content being worked on for the next iteration, Governance 1.1.
+
+### Addition of committees, subprojects, and working Groups
+
+TBD
```

Comment:
May I suggest noting which SIG or body has responsibility to decide these sections?

Reply:
That is an _excellent_ suggestion, as I had intended that to be clear in the governance and this is a great place to make that clear.
```suggestion
It is the purview of the initiating Community SIG to manage the Governance 1.1 process, including the responsibility to decide which types of groups get added, how they are created and ended, and where future responsibility for these type of decisions shifts once Governance 1.1 is ratified.
```